### DISCUSSION TITLE
How do I configure Lidar scripts to import albums within folders?

### DISCUSSION THREAD

**Discussion by wmazoni** [RandomNinjaAtk/arr-scripts#258] [Q&A] [answered]
I've already tried removing the settings from downloading the .lrc file in tidal-dl, but whenever I restart the pod the json file returns to the default configuration. My music is imported directly into the artist's root folder, I would like them to be imported into folders separated by albums, in the artist's folder, in another old installation that's how it worked. How can I adjust this?

It is currently importing like this
![2024-05-15_09-47](https://github.com/RandomNinjaAtk/arr-scripts/assets/41965426/1fde1a7f-2722-4047-b656-84115d144bf4)

I wish it were like that.
![2024-05-15_09-48](https://github.com/RandomNinjaAtk/arr-scripts/assets/41965426/6a17b90f-5065-43c4-8632-e15a86e144e6)

**Comment by RandomNinjaAtk** [comment]
You need to change lidarrs track naming settings to do that. Which is unrelated to these scripts

**Reply by wmazoni** [reply]
Would this configuration be in the beets-config-lidarr.yaml file?

**Comment by jaycedk** [comment]
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/17232146/ad16e91d-75c9-4b18-86aa-4f5fd17f7bdf)

**Reply by wmazoni** [reply]
Thanks.  I needed to leave Lidarr managing the files, this was disabled, now there is something missing that I couldn't adjust, even changing tidal-dl settings it is always downloading song lyrics (lrc files), how do I remove this option ?

**Reply by jaycedk** [reply]
Try checking you settings.
And remove .lic 
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/17232146/924d663d-231e-404e-84a5-04863b383535)

**Reply by jaycedk** [reply]
You can always check https://wiki.servarr.com/en/lidarr

**Reply by wmazoni** [reply]
When the container restarts it loses the lidar settings and I have to do the procedure again to adjust the folder organization. Is there a way for these checks to persist even after restarting the container?

**Reply by jaycedk** [reply]
Did you even go through your extended.conf and edit it to your liking ?
In there you can set it up like you want.

In you case you would want to set this line to false
configureMediaManagement="false"

**Reply by wmazoni** [reply]
I left this option as false, but whenever the pod restarts it continues to change the folder configuration I am configuring? Do I need to configure anything else to persist this configuration?

##### AUTOCONFIG SCRIPT SETTINGS
configureMediaManagement="false"               # Configures Lidarr Media Management Settings
configureMetadataConsumerSettings="false"      # Configures Lidarr Metadata Consumer Settings
configureMetadataProviderSettings="false"      # Configures Lidarr Metadata Provider Settings
configureCustomScripts="true"                  # Adds custom scripts to Lidarr
configureLidarrUiSettings="true"              # Configures Lidarr UI Settings
configureMetadataProfileSettings="false"       # Configures Lidarr Metadata Profile Settings
configureTrackNamingSettings="true"           # Configures Lidarr Track Naming Settings

**Reply by jaycedk** [reply]
Not that i know of.
Is your extended.conf being overridden ?

**Reply by RandomNinjaAtk** [reply, marked as answer]
@wmazoni if you read the settings descriptions, you would see that you need to disable the following setting by changing it to false:
`configureTrackNamingSettings="false" # Configures Lidarr Track Naming Settings`

You can see that the setting your complaining about falls under Track Naming in the GUI......

**Reply by wmazoni** [reply]
Thanks. It worked! Lidar is now saving the name configuration I added manually
